### PR TITLE
slingshot: update README

### PR DESCRIPTION
**Pull request by oleganza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +24,31 @@ based on the [Bulletproofs](https://doc.dalek.rs/bulletproofs/index.html) zero-k
...
 
 A _key blinding scheme_ for deriving hierarchies of public keys for [Ristretto](https://ristretto.group)-based signatures.
 
 * [Specification](keytree/keytree.md)
 
+### [Token](token)
+
+An API for issuing assets using ZkVM.
+
+
```
Comment (tessr) [inline README.md]: nit but I think you have an extra newline here

Comment (oleganza) [reply README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -34,11 +24,30 @@ based on the [Bulletproofs](https://doc.dalek.rs/bulletproofs/index.html) zero-k
 * [Spacesuit README](spacesuit/README.md)
 * [Cloak specification](spacesuit/spec.md)
 
+### [Musig](musig)
+
+A pure Rust implementation of the [MuSig signature aggregation scheme](https://eprint.iacr.org/2018/068) by Maxwell, Poelstra, Seurin and Wuille.
```
Comment (cathieyun) [inline README.md]: Let's keep the capitalization consistent - so maybe call it "Musig" or just "Simple Schnorr Multi-Signatures", as in the paper title

Comment (oleganza) [reply README.md]: fair

**Pull request merged**